<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_add">
       <attribute name="title">
        <string>+</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuMais_au_moins_a_remplit">
     <property name="title">
      <string>Mais au moins ça remplit</string>
     </property>
     <addaction name="actionWow"/>
    </widget>
    <addaction name="actionRien"/>
    <addaction name="actionEncore_rien"/>
    <addaction name="menuMais_au_moins_a_remplit"/>
    <addaction name="separator"/>
    <addaction name="actionEt_c_est_d_j_a"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAdieu"/>
    <addaction name="actionL_aide"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWow"/>
   <addaction name="actionRien"/>
   <addaction name="separator"/>
   <addaction name="actionEncore_rien"/>
   <addaction name="actionAdieu"/>
  </widget>
  <action name="actionRien">
   <property name="text">
    <string>Rien</string>
   </property>
  </action>
  <action name="actionEncore_rien">
   <property name="text">
    <string>Encore rien</string>
   </property>
  </action>
  <action name="actionAdieu">
   <property name="text">
    <string>Adieu</string>
   </property>
  </action>
  <action name="actionL_aide">
   <property name="text">
    <string>L'aide</string>
   </property>
  </action>
  <action name="actionEt_c_est_d_j_a">
   <property name="text">
    <string>Et c'est déjà ça</string>
   </property>
  </action>
  <action name="actionWow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wow</string>
   </property>
  </action>
  <action name="actionAjouter_un_tab">
   <property name="text">
    <string>Ajouter un tab</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
